packages/core/src/Stores/ui-store.js

```diff
@@ -110,6 +110,7 @@ export default class UIStore extends BaseStore {
     @observable should_show_toast_error = false;
     @observable mobile_toast_error = '';
     @observable mobile_toast_timeout = 1500;
+    @observable is_trade_params_expanded = true;
```

Comment:
Can we move this to `trade-store` instead? Since it's only relevant to DTrader.

Reply:
Sure 👍